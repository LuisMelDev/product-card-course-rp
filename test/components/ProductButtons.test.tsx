import React from 'react';
import renderer from 'react-test-renderer';
import { ProductButtons } from '../../src/components';

describe('it', () => {
  it('should render correcly', () => {
    const wrapper = renderer.create(<ProductButtons />);
    expect(wrapper).toMatchSnapshot();
  });
});
