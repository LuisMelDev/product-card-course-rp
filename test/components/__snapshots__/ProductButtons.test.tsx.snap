// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render correcly 1`] = `
<div
  className="buttonsContainer undefined"
>
  <button
    className="buttonMinus"
    onClick={[Function]}
  >
     
    -
     
  </button>
  <div
    className="countLabel"
  >
     
     
  </div>
  <button
    className="buttonAdd false "
    onClick={[Function]}
  >
     
    +
     
  </button>
</div>
`;
